### PR TITLE
Bug 1486504 - Navigation for Translate.Next

### PR DESCRIPTION
**Pull request by adngdb**
This adds a very simple navigation element at the top. It's a breadcrumb that allows to go back to the home page, to the team page, or to the project dashboard for the current team. Note that this PR doesn't close the bug, as I guess this is insufficient for our users.

### CODE REVIEW

File: frontend/src/App.css [hunk 1]
```diff
@@ -1,20 +1,25 @@
 #app {
-    display: flex;
-    height: 100%;
+    display: grid;
```
Comment (mathjazz) [inline frontend/src/App.css]: We dropped the idea of using `CCS grid` a while ago, because it's not supported by our browser support matrix.

Comment (adngdb) [reply frontend/src/App.css]: Can we maybe align our support matrix on the grid feature? I don't remember how we chose those browsers to support, so maybe that's a dumb question. :-)

Comment (mathjazz) [reply frontend/src/App.css]: OK, I'll have a look and PR the branch with an update to the CSS file. Or the browser matrix. :-)

**Comment by adngdb** [comment]
@mathjazz, I'd like to merge this now, because there are some refactoring in this that will impact my other work on T.N. Do you mind if we just merge now and make the needed changes in follow-up PRs?

**Comment by mathjazz** [comment]
I updated the styling but left CSS Grid in. So I also updated the Browser Support Matrix.

CSS is quite different from the one we used in Translate.Current, so the changeset would be quite extensive. Besides, we'll probably use more CSS Grid in the future.

**Comment by adngdb** [comment]
Hurray for Grid!

**Pull request merged**